Welcome SFTP Client (#39676)
home-assistant/home-assistant.github.io: source/_integrations/sftp_client.markdown

---
title: SFTP Client
description: Instructions on how to use  SFTP Client in Home Assistant.
ha_category:
  - Backup
ha_iot_class: Local Polling

> ha_release: '2025.2'

Reviewer: ```suggestion
ha_release: '2025.8'
```

ha_config_flow: true
ha_domain: sftp_client
ha_codeowners:
  - '@cyr-ius'
ha_integration_type: service
ha_quality_scale: platinum
related:
  - docs: /common-tasks/general/#backups
    title: Creating backups in Home Assistant
---

> This {% term integration %} allows you to perform Home Assistant backups to a server via SFTP. Once this integration is configured, your server will contain a new folder called backup where all backups will be stored. If you delete the folder, it will be automatically recreated as long as the {% term integration %} is enabled.

Reviewer: ```suggestion
The **SFTP Client** {% term integration %} allows you to perform Home Assistant backups to a server via <abbr title="secure file transfer protocol">SFTP</abbr>. Once this integration is configured, your server contains a new folder called backup where all backups will be stored. If you delete the folder, it will be automatically recreated as long as the {% term integration %} is enabled.
```

> 

Reviewer: ```suggestion

## Known limitations

- The integration can only access files that it creates in the Home Assistant folder. It cannot access or modify any other files.

```

## Removing the integration

{% include integrations/remove_device_service.md %}

If you remove the integration, the Home Assistant folder on your server is not automatically deleted. You must delete it manually.

> ## Known limitations
> 
> - The integration can only access files that it creates in the Home Assistant folder. It cannot access or modify any other files.
> 

Reviewer: ```suggestion
```

---
title: SFTP Client
description: Instructions on how to use  SFTP Client in Home Assistant.
ha_category:
  - Backup
ha_iot_class: Local Polling
ha_release: '2025.8'
ha_config_flow: true
ha_domain: sftp_client
ha_codeowners:
  - '@cyr-ius'
ha_integration_type: service
ha_quality_scale: platinum
related:
  - docs: /common-tasks/general/#backups
    title: Creating backups in Home Assistant
---

The **SFTP Client** {% term integration %} allows you to perform Home Assistant backups to a server via <abbr title="secure file transfer protocol">SFTP</abbr>. Once this integration is configured, your server contains a new folder called backup where all backups will be stored. If you delete the folder, it will be automatically recreated as long as the {% term integration %} is enabled.

> 

Reviewer: ```suggestion
{% include integrations/config_flow.md %}

```
